src/bokeh/util/deprecation.py

```diff
@@ -77,6 +82,29 @@ def deprecated(since_or_msg: Version | str,
 
     warn(message)
 
+def find_stack_level() -> int:
+    """
+    Find the first place in the stack that is not inside Bokeh.
```

Comment:
nit: all the docstings in the repo start like this:
```suggestion
    """ Find the first place in the stack that is not inside Bokeh.
```